src/api/lambda.ts

```diff
@@ -53,14 +53,20 @@ export const lambdaCall = (fnName: string, payload: any): Promise<any> => new Pr
 
             resolve(body);
           } catch(e) {
-            logger.error('Erroed parsing response body: ', data);
+            logger.error(`Erroed parsing response body: ${data}`);
 
             return reject(e.message);
           }
         }
       });
 });
 
+export const invokeReorg = async (): Promise<ApiResponse> => {
+  const response = await lambdaCall('onHandleReorgRequest', {});
+
+  return response;
```

Comment:
Will we need the response for anything? If not, shouldn't we just return the promise `lambdaCall(...)`?

Reply:
I agree. Done in f5bc6f4184d2ad6a86c8cd31f5d91ae927a7652a